tstrait/genetic_value.py

```diff
@@ -0,0 +1,184 @@
+import numba
+import numpy as np
+import pandas as pd
+import tskit
+
+
+@numba.njit
+def _traversal_genotype(
```

Comment:
is this the same function as in `simulate_phenotype.py`?

Reply:
Yes, it is, and I'm planning to remove it from `simulate.phenotype.py`.